CMakeLists.txt

```diff
@@ -834,28 +834,28 @@ if(BUILD_CLI)
   if(UNIX)
      if(APPLE)
        if (ARCH MATCHES arm64)
-         set(OPENSTUDIO_GEMS_ZIP_FILENAME "openstudio3-gems-20230918-darwin_arm64.tar.gz")
-         set(OPENSTUDIO_GEMS_ZIP_EXPECTED_MD5 "404783d585a63a07f2f9d5b4ba445d99")
+         set(OPENSTUDIO_GEMS_ZIP_FILENAME "openstudio3-gems-20231011-darwin_arm64.tar.gz")  # TODO: redo
+         set(OPENSTUDIO_GEMS_ZIP_EXPECTED_MD5 "a98520ab33fa60aee008adec9ba7a999")
        else()
-         set(OPENSTUDIO_GEMS_ZIP_FILENAME "openstudio3-gems-20230918-darwin.tar.gz")
-         set(OPENSTUDIO_GEMS_ZIP_EXPECTED_MD5 "71282202360c68cf9dd0f0d585d560f2")
+         set(OPENSTUDIO_GEMS_ZIP_FILENAME "openstudio3-gems-20231014-darwin.tar.gz")
+         set(OPENSTUDIO_GEMS_ZIP_EXPECTED_MD5 "d37f422e6d5792f2646de9f03ed31d94")
        endif()
      else()
        if (ARCH MATCHES "arm64")
          set(OPENSTUDIO_GEMS_ZIP_FILENAME "openstudio3-gems-20230918-linux_arm64.tar.gz")
```

Comment:
todo

Reply:
Since the PR is mergede, revert the todo.